packages/scatterplot/src/props.js

```diff
@@ -60,6 +60,8 @@ export const ScatterPlotPropTypes = {
     symbolSize: PropTypes.oneOfType([PropTypes.func, PropTypes.number]).isRequired,
     symbolShape: PropTypes.oneOfType([PropTypes.oneOf(['circle', 'square'])]).isRequired,
 
+    symbol: PropTypes.func,
```

Comment:
What about canvas implementation? Also, if we use something like `memo()` to wrap the symbol component (which should be the case IMO), you'll have an object not a function so we should support both types

Reply:
The TypeScript definition should be updated too.

Reply:
Both function and object are allowed, will do the TS def soon.